strawberryfields/program_utils.py

```diff
@@ -391,3 +393,80 @@ def lex_topo(seq, key):
     B = C[:ind]  # marked and still possibly unmarked
     C = C[ind:]  # final unmarked instances
     return A, B, C
+
+
+def optimize_circuit(seq):
+    """Try to simplify and optimize a quantum circuit.
+
+    The purpose of the optimizer is to simplify the circuit
+    to make it cheaper and faster to execute. Different backends may require
+    different types of optimization, but in general the fewer operations a circuit has,
+    the faster it should run. The optimizer thus should convert the circuit into a
+    simpler :term:`equivalent circuit`.
```

Comment:
Sphinx is giving a warning that `equivalent circuit` is not found in the glossary

Reply:
It will be once the glossary PR is merged :)